example/lib/main.dart

```diff
@@ -51,20 +55,23 @@ void main() {
 ///Initialize function is always called at the start of your application or within
 ///the scope of use of LexActivator
 void initializeLexActivator() {
-  LexActivator.SetProductData(productData: "PASTE_CONTENT_OF_PRODUCT.DAT_FILE");
+  LexActivator.SetProductData(productData: "SET_PRODUCT_DATA");
 
   LexActivator.SetProductId(
-      productId: "PASTE_PRODUCT_ID", flags: LexActivator.LA_USER);
+      productId: "SET_PRODUCT_ID", flags: LexActivator.LA_IN_MEMORY);
```

Comment:
@mumin-khan In example we use LA_USER permission flag by default as that is the most common use case.